### PR TITLE
Installer fixes

### PR DESCRIPTION
**Pull request by cosminbosutar**
Moved api to extension folders.
Fixed security check based on user_token instead of api_id.
Fix customer identifier filter issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,69 @@
...
+
+All payments will be processed in a secure PCI DSS compliant environment so you don't have to think about any such compliance requirements in your web shop.
+
+Install
+=======
+
+### Automatic
+1. Download the Twispay payment module from Opencart Marketplace, where you can find [The Official Twispay Payment Gateway Extension][marketplace]
+
+2. Use the Opencart 3 Extension Installer and upload the OCmod ar
```
Comment (dragos-ursu) [inline README.md]: Please review the description on this line.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,69 @@
...
+
+3. Sign in to your OpenCart admin.
+
+4. Click **Extensions** tab and **Payments subtab**.
+
+5. Under **Twispay** click **Install** and then click **Edit**.
+
+6. Select **Enabled** under **Status**.
+
+7. Select **No** under **Test Mode**. _(Unless you are tesing)_
```
Comment (dragos-ursu) [inline README.md]: Please correct typo `tesing`.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,69 @@
...
+* Initial Plugin version
+
+<!-- Other Notes
+===========
+
+A functional description of the extension can be found on the [wiki page][doc] -->
+
+[twispay]: http://twispay.com/
+[marketplace]: https://www.opencart.com/index.php?route=marketplace/extension/info&extension_id=31761&filter_member=twispay
+[github]: https://github.com/MichaelRotaru/OpenCart3.0
```
Comment (dragos-ursu) [inline README.md]: Please link to `https://github.com/Twispay/OpenCart3.0`.

**Review by dragos-ursu — CHANGES_REQUESTED** [review]
Please check the comments.

File: README.md (hunk 1 above)
Comment (cosminbosutar) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (cosminbosutar) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (cosminbosutar) [reply README.md]: Done

**Review by dragos-ursu — APPROVED** [review]
Code review done

**Pull request merged**